### DISCUSSION TITLE
galaxy projector

### DISCUSSION THREAD

**Discussion by many7695** [milo526/homebridge-tuya-web#458] [General]
i have a Galaxy Projector, Star Projector Smart Night Light Works with Alexa, Google Assistant if from galaxycove, is there a way this cab be added to my devices a long with a power strip both of them work on the smartlife app but dont show up in accesories

**Comment by malwcal** [comment]
There is a tuya plugin that has been customised to work with a galaxy projector.  It is called homebridge-star-projector [https://github.com/SeydX/homebridge-star-projector#readme](url)

It works with my galaxy project sold in Australia by kogan - maybe it works with yours as well?
